tests/js/spec/utils/useMetricsContext.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {MetricsProvider} from 'sentry/utils/metrics/metricsProvider';
+import {useMetricsContext} from 'sentry/utils/useMetricsContext';
+
+function TestComponent({other}: {other: string}) {
+  const {metricTags, metricMetas} = useMetricsContext();
+  return (
+    <div>
+      <span>{other}</span>
+      {metricTags &&
+        Object.entries(metricTags).map(([key, tag]) => <em key={key}>{tag.key}</em>)}
+      {metricMetas &&
+        Object.entries(metricMetas).map(([key, meta]) => <em key={key}>{meta.name}</em>)}
+    </div>
+  );
+}
+
+describe('useMetricsContext', function () {
+  const {organization, project} = initializeOrg();
+  let metricsTagsMock: jest.Mock | undefined = undefined;
+  let metricsMetaMock: jest.Mock | undefined = undefined;
+
+  beforeEach(function () {
+    metricsTagsMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/metrics/tags/`,
+      body: [{key: 'environment'}, {key: 'release'}, {key: 'session.status'}],
+    });
+
+    metricsMetaMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/metrics/meta/`,
+      body: [
+        {
+          name: 'sentry.sessions.session',
+          type: 'counter',
+          operations: ['sum'],
+        },
+        {
+          name: 'sentry.sessions.session.error',
+          type: 'set',
+          operations: ['count_unique'],
+        },
+      ],
+    });
+  });
+
+  it('works', async function () {
```

Comment:
Works is big word that can mean many things :D can we be more specific here? Maybe `fetches metrics and tags`?

Reply:
:D yes of course